src/custom/components/Header/MobileMenuIcon/index.tsx

```diff
@@ -0,0 +1,82 @@
+import styled, { css } from 'styled-components/macro'
+
+const Wrapper = styled.div<{ isMobileMenuOpen: boolean; height?: number; width?: number; lineSize?: number }>`
+  z-index: 102;
```

Comment:
102 😬😬😬

Reply:
Yeah I know...I'd want to tackle the z-index thing in a separate PR. I think moving it to a const file, and then defining some general levels would be better.